src/User32.Desktop/User32.cs

```diff
@@ -144,6 +144,17 @@ public static partial class User32
         [DllImport(nameof(User32))]
         public static extern unsafe IntPtr SendMessage(IntPtr hWnd, int wMsg, void* wParam, void* lParam);
 
+        [DllImport(nameof(User32))]
+        public static extern unsafe IntPtr SendMessage(IntPtr hWnd, WindowMessage wMsg, void* wParam, void* lParam);
```

Comment:
Sadly, I think implicit casts to and from enums can't be defined (at least not in C#). I tried hard for the error code structs/enums and never found a way. 
I don't object to `WindowMessage` as an enum being used though instead of `int`. We often define enums and use them instead of `int` where the values can be (largely) defined. Folks can pass in custom values by casting any int to the enum type. My only regret here is having both overloads. So it seems to me this should be a Helper function instead of another P/Invoke. 

One benefit of having just one P/Invoke method and using Helpers for the rest is that when we have to make a fix to the P/Invoke (say, we forgot to add `SetLastError=true`), we don't risk forgetting to update all the overloads, since there is only ever one of them.